### DISCUSSION TITLE
Yocto Image size

### DISCUSSION THREAD

**Discussion by Shivaraj199** [polarfire-soc/polarfire-soc-documentation#370] [Linux build systems for PolarFire SoC] [answered]
The images core-image-minimal-dev-cli and mpfs-dev-cli both .wic file  have an identical image size of 7.3 GB, which seems unusually large.The core-image-minimal-dev-cli and mpfs-dev-cli images should have different sizes, reflecting their intended minimal configurations.

**Comment by vfalanis** [comment]
Hi @Shivaraj199 ,

The core-image-minimal-dev-cli and mpfs-dev-cli partition sizes are currently setup to fit the full eMMC size (~7.3GB). We are currently working on adding support to automatically resize the partitions at runtime so that the WIC image file size is reduced. We will update this post once the support has been added. 

Thanks

**Reply by Shivaraj199** [reply]
@vfalanis Is there any updates in this?

**Reply by Shivaraj199** [reply]
Hello @vfalanis, 
is the support added?

**Reply by vfalanis** [reply, marked as answer]
Hi @Shivaraj199 ,

Yes this was added in the [v2024.02](https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp/releases) release "add support for growing the size of the root partition on first boot". 

I would suggest to try the v2024.02 PolarFire SoC Yocto release

Thanks

**Reply by Shivaraj199** [reply]
Thank you @vfalanis
